awx/main/redact.py

```diff
@@ -12,23 +10,13 @@ class UriCleaner(object):
 
     @staticmethod
     def remove_sensitive(cleartext):
-        # exclude_list contains the items that will _not_ be redacted
```

Comment:
There's not really anything to redact here (we don't support basic auth anymore).

Also, all our credentialing data is passed in via environment variables (a token).

Here's some stdout that illustrates that these URLs render just fine without this code:

![image](https://user-images.githubusercontent.com/214912/89330511-6223ba80-d65e-11ea-8b07-e6175eab24f6.png)

Reply:
I'm a little unclear why were doing this in the first place, @AlanCoding:

https://github.com/ansible/awx/commit/d59d8562db35a8b1d7769cc8081c84028779c931

The point of this function is to take URLs like:

`https://USER:PASS@foo`

...and turn them into `https://*********@foo`

Why would we ever want to whitelist Galaxy URLs to _prevent_ that behavior?  I don't think we need this code at all anymore, but I think it was flawed to start with.

Reply:
If it was important we would have test coverage for it. Maybe I was going crazy, I'm not sure.